### PR TITLE
Update README with instructions on how to add a special route

### PR DESCRIPTION
**Pull request by chao-xian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,31 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
...
+```
+
+To add a new entry:
+
+1. In a Rails console run
+  ```ruby
+  irb(main):001:0> SecureRandom.uuid
+```
+2. Use the result as the value for `content_id`
+3. Edit the `/data/special_routes.yaml` file with your new special route.
```
Comment (leenagupte) [inline README.md:35]: I wonder if these two sections can be consolidated?  For example:

---

Add an entry in [`/data/special_routes.yaml`](https://github.com/alphagov/special-route-publisher/blob/a74101c47fffd80123efbfd1d095398a40bdc594/data/special_routes.yaml), for example:

```
- :content_id: 'c1f08359-21f7-49c1-8811-54bf6690b6a3'
  :base_path: '/account/home'
  :title: 'Account home page'
  :rendering_app: 'frontend'
```
The value for the `content_id` can be generated by running `SecureRandom.uuid` in a rails console.

**Review by leenagupte — COMMENTED** [review]
It's great that you're updating these docs. Added a suggestion for shortening them a bit.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,24 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
...
+Add an entry to `/data/special_routes.yaml`, for example:
+
+```
+- :content_id: 'c1f08359-21f7-49c1-8811-54bf6690b6a3'
+  :base_path: '/account/home'
+  :title: 'Account home page'
+  :rendering_app: 'frontend'
+```
+
+You can generate a new value for `content_id` from running in the Rails console `SecureRandom.uuid`.
```
Comment (leenagupte) [inline README.md:28]: Suggestion:

You can generate a new value for `content_id` by running `SecureRandom.uuid` in a ruby console.

**Review by leenagupte — COMMENTED** [review]
One suggestion to make the sentence flow a bit better. Also I don't think this is a rails app so that could trip people up.

File: README.md (hunk 2 above)
Comment (chao-xian) [reply README.md]: 👍

**Review by leenagupte — APPROVED** [review]

**Pull request merged**